### ISSUE TITLE
Dashboard -> Statistics not update

### ISSUE THREAD

**Issue by skyhighcs** [opensupports/opensupports#190]
Hi,

Anyone please help

![statistic](https://user-images.githubusercontent.com/4660792/38994490-c4b2e122-4410-11e8-975e-5319d9135cc5.png)

![statistic2](https://user-images.githubusercontent.com/4660792/38994493-c64570c2-4410-11e8-94ab-64f5f01eb621.png)

**Comment by ivandiazwm** [comment]
Stats should be shown until the day before the current one. How much time has happened since the system was installed?

**Comment by ernestomartin** [comment]
I have te same issue . I installed Package version: opensupports_v4.1.3 in July 17 and the dashboard show 0 in all ticket stats.  
my server is apache with  PHP 5.6.30 & mysql 5.7 
Thanks  :)

**Comment by Th14go** [comment]
I have the same problem...
According to the developer, the results is after a few days, I can give an opinion, I believe it should be in real time, created ticket, already appears, closed already appears.

**Comment by mredigonda** [comment]
I agree that this should be updated in real time, however at this point, the system is designed to be updated after each day. I hope we will improve this soon.
What I want to confirm now is if the system has the expected behavior, does it update after each day?

**Comment by ernestomartin** [comment]
Thanks Maximiliano.

**Comment by pieter888** [comment]
I feel like one update every 24 hours is a way to large gap. Real time statistics would be way more usable. Or once every hour at least.

**Comment by nickbe** [comment]
Statistics in the current form are mere visual gags to me.

What would really be needed is an admin only view:

* Average assigned ticket per staff member over the last 6 weeks, 3 months, 1 year (three values)
* Users which deviate siginifcantly below or above average
* Average time to solve ticket sorted by priority.
* Average open tickets over the last 6 weeks, 3 months, 1 year (three values)

Also you‘d need a escalation time for each priority. Tickets should then for example be marked as critical (High prio - too much time) automatically

**Comment by pieter888** [comment]
> Average open tickets over the last 6 weeks, 3 months, 1 year (three values)
I fully agree with this.
Seeing the current amount of open tickets over time is a great way to see how the team is performing on closing them. 

Showing the amount of opened and closed tickets on any given day doesn't make much sense without seeing the overall picture for context. 

> Average time to solve ticket sorted by priority.
Yes, for project managers this is a hot stat. Can be use as a selling point for how good your support is too.

Most importantly, I want so see how many balls our team are juggling at the same time. Amount of open/unassigned issues over time would be a great metric.  
AFAIK this can be calculated from the current stats (total opened - total closed = total open tickets)
